e2e/scenario_test.go

```diff
@@ -334,6 +338,9 @@ func Test_marinerv2_azurecni(t *testing.T) {
 				nbc.ContainerService.Properties.AgentPoolProfiles[0].Distro = "aks-cblmariner-v2-gen2"
 				nbc.AgentPoolProfile.Distro = "aks-cblmariner-v2-gen2"
 			},
+			LiveVMValidators: []*LiveVMValidator{
+				kubeletNodeIPValidator(),
```

Comment:
any reason you explicitly added the new validation to these scenarios, rather than as a common validator?

Reply:
yes, it's because there's one test `Test_ubuntu2204privatekubepkg` that pins the k8s version to 1.25.6, so it won't have the node IP flag set.